new/LIT7250PPFanuel.xml

```diff
@@ -0,0 +1,76 @@
+                <title xml:lang="en" xml:id="t1">Protective stanza to Phanuel</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="JK"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A monorhymed five-line stanza to <ref type="pers" corresp="PRS7840Phanuel">the archangel Phanuel</ref>.
+                    In <ref type="mss" corresp="UppEt46"/>,
+                it is followed by an unrhymed but thematically connected line that ends abruptly.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Magic"/>
+                    <term key="Poetry"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2025-01-16">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7250PPFanuel" passive="UppEt46"/>
+                </listRelation>
+            </div>
+            <div xml:lang="gez" type="edition">
+                <note>Below, the full text is reproduced based on <ref type="mss" corresp="UppEt46"/>.</note>
+                <div type="edition">
+                    <ab>
+                        <l n="1">ነዓ፡ ኀቤየ፡ ፋኑኤል፡ እሳት፡ ነዳዲ፡</l>
+                        <l n="2">ማኅበረ፡ ሰይጣናት፡ በሰማይ፡ እስመ፡ አንተ፡ ሰዳዲ፡</l>
+                        <l n="3">አልብሰኒ፡ ጽድቀ፡ ለዓለመ፡ <sic resp="JK">ዓለመ፡</sic> ወዓዲ፡</l>
+                        <l n="4">ወበትንባሌከ፡ እደ። ኃጣውእየ፡ ፍዲ፡</l>
+                        <l n="5">መናሥግተ፡ ቤትየ፡ አጽንዕ፡ ኢይባእ፡ ረዋዲ፡</l>
+                        ኦአምላከ፡ ፋኑኤል፡ ዕቀባ፡ እምዓይነተ፡ ባርያ፡ ወ
```

Comment:
and the last line is not a line?

Reply:
It's not rhymed and, as we know, such verses usually consist of five lines, so not marking it up as `<l n="6">` was a conscious decision. But do you think it will look weird like this?

Reply:
I'll mark it up as Line 6 for the sake of the encoding – it doesn't really make any difference contentwise.